bin/copyRawDataToPrmTiming.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 set -u
 
-# module load NGS_Automated/beta-bare; copyRawDataToPrmTiming.sh -g umcg-atd -s gattaca01.gcc.rug.nl -r /groups/umcg-atd/scr01/ -l DEBUG
+# module load NGS_Automated/beta-bare; copyRawDataToPrmTiming.sh -g umcg-atd -s gattaca02.gcc.rug.nl -r /groups/umcg-atd/scr01/ -l DEBUG
```

Comment:
There should not be any hard-code values for group names nor of machines in the scripts. I assume this line is meant as an example of how to run/execute this script, but that would preferably go into a command line help option. E.g. ```script_name.sh -h``` or when the script is executed without or with the wrong number of arguments you can print an example of what the script is expecting...